strawberry/extensions/directives.py

```diff
@@ -18,7 +18,7 @@ async def resolve(
     ) -> AwaitableOrValue[Any]:
         result = await await_maybe(_next(root, info, *args, **kwargs))
 
-        for directive in info.field_nodes[0].directives:
+        for directive in info.field_nodes[0].directives:  # type: ignore
```

Comment:
Do we ignore the type here because `info.field_nodes` now is a `Collection[FieldNode]` and we can't be sure `info.field_nodes[0]` is actually a single `FieldNode`? Maybe we should ask @Cito why they went with `Collection[FieldNode]` instead of `List[FieldNode]` 🤔

Reply:
Good question @DoctorJohn. I think it was a mistake, and reverted the change in the main branch of graphql-core already.

Reply:
Thanks @Cito for responding and thanks @DoctorJohn for reviewing, will update the PR soon :)